### ISSUE TITLE
Full screen broadcast not working after first start

### ISSUE THREAD

**Issue by maniaque** [epoptes/epoptes#116]
Found strange issue: full screen broadcast not working when I provision clean VM with epoptes and start epoptes for the first time.

Re-run it from terminal shows up the following error on full screen broadcast:
```storepasswd: No such file or directory
-storepasswd failed for file: /home/teacher01/.config/epoptes/vncpasswd
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/epoptes/ui/gui.py", line 326, in on_imi_broadcasts_broadcast_screen_fullscreen_activate
    self.broadcast_screen('true')
  File "/usr/lib/python3/dist-packages/epoptes/ui/gui.py", line 308, in broadcast_screen
    with open(pwdfile, 'rb') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/home/teacher01/.config/epoptes/vncpasswd'
```

Looks like `.config/epoptes` directory is missing after the first start, but when I restart epoptes once again it is already created.

I've fixed this in my provision scripts by creating `~/.config/epoptes` with standard 755 permissions, but looks like there is something to fix in code :)

**Comment by alkisg** [comment]
Hi, which epoptes version is this? `dpkg -l epoptes`

**Comment by maniaque** [comment]
Already destroyed the VM, but it was on Ubuntu 18.04, latest version from stable ppa.

**Comment by maniaque** [comment]
If it helps, this is a first login and first launch of epoptes. Looks like epoptes creates `~/.config/epoptes` when exits first time (maybe saving settings or something else). And issue appears only at first start (or when you recreate user from scratch).

**Comment by alkisg** [comment]
Thanks, I reproduced it in the latest Epoptes version, I will fix it in the next release.

**Comment by alkisg** [comment]
Thanks, fix committed in c24fd41.
This also addresses a possible problem of .config getting created with mode!=0o700 if it didn't exist at all.
